catkit/emulators/ZwoCamera.py

```diff
@@ -79,6 +82,10 @@ def set_image_type(self, image_type):
     def capture(self, initial_sleep=0.01, poll=0.01, buffer=None, filename=None):
         pass
 
+    @abc.abstractmethod
```

Comment:
[revision] This needs to be removed. It doesn't make sense to be abstract _and_ have a def.

Reply:
Agreed. I forgot to remove that. Sorry.